adal/build/LayoutNugetPackage.proj

```diff
@@ -1,290 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is too complex - we don't need to list every file that gets copied, it is sufficient to copy the output.